scene/3d/area_3d.cpp

```diff
@@ -172,9 +172,11 @@ void Area3D::_initialize_wind() {
 
 	// Overwrite with area-specified info if available
 	if (!wind_source_path.is_empty()) {
-		Node3D *p_wind_source = Object::cast_to<Node3D>(get_node(wind_source_path));
-		ERR_FAIL_NULL(p_wind_source);
-		Transform3D global_transform = p_wind_source->get_transform();
+		Node *wind_source_node = get_node_or_null(wind_source_path);
+		ERR_FAIL_NULL_MSG(wind_source_node, "Area3D: Path to wind source is invalid: " + wind_source_path + ".");
```

Comment:
Prefixes seem excessive, you already see which file it's in in the error trace. Do we follow this pattern anywhere else?

Reply:
Copied from `Viewport`, will remove, should I remove in `Viewport` as well?

Reply:
I'd say yes, it was added in [this _groundbreaking_ commit](https://github.com/godotengine/godot/commit/a503f8aadcb8cbc85bde589fb25ea06e599b367b) long time ago, so it likely predates any attempts at unifying error messages.

Reply:
There's a case to be made for adding the class name automatically to error messages. While it can be inferred from the source line, it's not necessarily obvious for people not familiar with the engine codebase (and can be lacking context in the case of a ViewportTexture error defined in `viewport.cpp`).